app-editors/emacs/emacs-29.2-r2.ebuild

```
@@ -586,6 +586,14 @@ src_install() {
 			\"${EPREFIX}/Applications/Gentoo\". You may want to copy or
 			symlink it into /Applications by yourself."
 	fi
+	if ! use mailutils; then
+		DOC_CONTENTS+="The mailutils USE flag is disabled.\\n If emacs'
+		own email features are going to be used as an email client
+		(rmail, gnus), you are strongly encouraged to enable it. If not,
+		Emacs will use its own implementation of movemail; which has
+		less features and is less secure. \\n For more information see:
+		https://www.gnu.org/software/emacs/manual/html_node/emacs/Movemail.html"
```

1. Add a double newline at the beginning, so this block of text will be separated from the one preceding it.
2. Avoid newlines in the text (readme.gentoo.eclass will do line breaking itself).
3. As @xgqt mentioned before, spellings should be "Emacs" and "e-mail".
4. "Features" is a countable noun, so it's "fewer features".

```suggestion
		DOC_CONTENTS+="\\n\\nThe mailutils USE flag is disabled. If Emacs'
		own e-mail features are going to be used as an e-mail client
		(e.g. Rmail), you are strongly encouraged to enable it. If not,
		Emacs will use its own implementation of movemail; which has
		fewer features and is less secure. For more information see:
		https://www.gnu.org/software/emacs/manual/html_node/emacs/Movemail.html"
```

---

Noted. I forced these changes to the latest commit so that it all remained in a single commit.

Thank you very much for the feedback

> "Features" is a countable noun, so it's "fewer features".

If my english teacher were to find out I made that mistake he would kill me :see_no_evil: